src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -200,6 +200,43 @@ WorkspaceServicePositionColumnOutOfRange(int line) = Position is outside of colu
 WorkspaceServiceBufferPositionOutOfOrder(int sLine, int sCol, int eLine, int eCol) = Start position ({0}, {1}) must come before or be equal to the end position ({2}, {3})
 
 ############################################################################
+# Edit Data Service
+
+EditDataSessionNotFound = Edit session does not exist.
+
+EditDataUnsupportedObjectType(string typeName) = Database object {0} cannot be used for editing.
+
+EditDataQueryNotCompleted = Query has not completed execution
```

Comment:
A lot of these definitely are edge cases and very unlikely to occur. I definitely think we should be catching these edge cases -- assuming they won't happen will lead to disaster and likewise throwing generic "failed" messages will make things a pain to track down. Maybe the wording of the errors can be changed or we can have more detailed errors for internal usage, but I think that can wait for a later PR.